packages/editor/playground/playground.tsx

```diff
@@ -173,21 +136,15 @@ const App: FC = () => {
             onContentSave={setContent}
         />
     )
+    const debugConsole = (
+        <DebugConsole hasUnsavedChanges={hasUnsavedChanges} contentId={contentId} content={content} setContentId={setContentId} />
+    )
```

Comment:
```suggestion
    const debugConsole = enableDevTools ? (
        <div className="playground-self-scroll"><DebugConsole hasUnsavedChanges={hasUnsavedChanges} contentId={contentId} content={content} setContentId={setContentId} /></div>
    ) : null 
```

When `enableDevTools` is false, we should not render `DebugConsole`